mm2src/coins/eth.rs

```diff
@@ -438,7 +440,7 @@ pub struct EthCoinImpl {
     /// the block range used for eth_getLogs
     logs_block_range: u64,
     nonce_lock: Arc<AsyncMutex<()>>,
-    erc20_tokens_infos: Arc<Mutex<HashMap<String, Erc20TokenInfo>>>,
+    erc20_tokens_infos: Arc<Mutex<BTreeMap<String, Erc20TokenInfo>>>,
```

Comment:
Could you tell why you decided to replace `HashMap` with `BTreeMap`?

Is the ordering of tokens critical?
If yes, could you provide the code place where it is more preferable for us?

Just get_all_balance_results_concurrently implies that that the order of balance results doesnt matter.

Reply:
It was different initially, I forgot to revert this part. Thanks!